game.py

```diff
@@ -0,0 +1,105 @@
+from collections import defaultdict
+
+from etc.utils import Color, read_words, get_random_word, is_valid_word
+from etc.consts import MAX_ATTEMPTS, INDEX_NOT_FOUND
+
+
+class WordleGame:
+    _WORDS = read_words()
```

Comment:
move to __init__

Reply:
the initialization, the variable or both?

Reply:
The initialization should be in the __init__, so you can just move both